src/ripple/app/main/Application.cpp

```diff
@@ -931,20 +932,42 @@ class ApplicationImp
         {
             auto j = logs_->journal("NodeObject");
             NodeStore::DummyScheduler scheduler;
+            /** DB instances require a parent/stoppable argument. However, this
+             * source instance is special - it's a short-lived (block scope)
+             * read-only instance used for import, and it has zero read threads.
+             * If we were to construct it with one of the main app stoppables
+             * (like jobqueue), then this temporary DB instance is left as a
+             * dangling child in the stoppable tree (because there is no way to
+             * unregister an object as a child).
+             *
+             * SOLUTION: create a dummy stoppable tree/parent for this
+             * particular source DB instance because it has nothing that needs
+             * stopping anyhow.
+             **/
```

Comment:
I don't think the comment is necessary as `dummyRoot` is self explanatory but it is up to you.

Reply:
Consider renaming `schedule` to `dummySchedule`

Reply:
done (both) - thx